docs/modules/ROOT/pages/tutorial-onpremise.adoc

```diff
@@ -1,145 +1,120 @@
+.Couchbase cluster overview image
+image::couchbase-cluster-overview-vm.png[]
 
-==== Alerting rules
+==== Alerts
 
-We want to be able to override in two specific ways:
-. Extend: provide Couchbase default rules and add custom ones.
-. Replace: only provide custom rules.
+CMOS comes with pre-installed alert rules to monitor the Couchbase cluster.
+Navigate to Alertmanager, prometheus UI to check the alert rules and alerts.
+For more information check the prometheus and alerting configuration section.
 
-To support this there are two locations to be used:
-. `/etc/prometheus/alerting/custom/`
-. `/etc/prometheus/alerting/`
+.Alert rules image
+image::prometheus-alert-rules.png[]
 
-The first only adds whilst the second will replace the defaults.
-Of course you can also just specify files rather than the full directory which can be useful to inhibit default rules by removing them from the file.
+.Alerts image
+image::prometheus-alerts.png[]
 
-We also support tuning of rules by environment variable.
-When Prometheus is launched, it will run `envsubst` on the files with all available environment variables then substituted.
-Be aware that if you use a variable then you cannot currently default it other than providing that default to the link:https://github.com/couchbaselabs/observability/blob/main/microlith/entrypoints/prometheus.sh[entrypoint^] script for substitution: a variable must be defined.
+==== Loki
 
-=== Alert manager
+Loki, which is shipped with Grafana, allows access to logs of various components.
+You can configure it via Configuration > Data sources > Loki > Explore.
 
-From a Prometheus perspective we always assume there is a local Alert Manager target so this may report as failed if disabled.
+.Loki explore dashboard image
+image::loki-explore-dashboard.png[]
 
-Additional alert managers can be specified by adding using the same `<file_sd_config>` link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#file_sd_config[syntax^] to the `/etc/prometheus/alertmanager/custom/` directory.
+From the Log browser, you can enter a custom Loki query or select appropriate labels to see the logs.
 
-=== Cluster Monitor
-The cluster manager will auto-start and configure itself with credentials supplied via the following environment variables:
+.Loki log browser image
+image::loki-log-browser.png[]
 
-* `CLUSTER_MONITOR_USER=${CLUSTER_MONITOR_USER:-admin}`
-* `CLUSTER_MONITOR_PWD=${CLUSTER_MONITOR_PWD:-password}`
-* `CLUSTER_MONITOR_ENDPOINT=${CLUSTER_MONITOR_ENDPOINT:-http://localhost:7196}`
+After that select the “Show logs” to view logs.
+You can also build  https://grafana.com/docs/grafana/latest/getting-started/getting-started/#step-3-create-a-dashboard[custom Grafana dashboards^] based on your needs.
```

Comment:
```suggestion
You can also build  https://grafana.com/docs/grafana/latest/getting-started/getting-started/#step-3-create-a-dashboard[custom Grafana dashboards^] based on your needs.

[NOTE]
====
You will need to install and configure Fluent Bit on your Couchbase Server nodes before logs will be sent to Loki.
Refer to the xref:deployment-fluentbit.adoc[Fluent Bit install guidance].
====
```

IMHO worth calling out since this isn't the default behaviour (if you think this admonition should be elsewhere in this section feel free to move it).